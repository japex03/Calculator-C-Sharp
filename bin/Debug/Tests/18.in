sin(3.14/6)^-11