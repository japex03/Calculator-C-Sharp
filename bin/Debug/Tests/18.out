2058,3868884